@here/harp-webpack-utils/scripts/HarpWebpackConfig.js

```diff
@@ -4,29 +4,45 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/*
 import * as CopyWebpackPlugin from "copy-webpack-plugin";
 import * as HtmlWebpackPlugin from "html-webpack-plugin";
+import * as path from "path";
 import { Configuration, Plugin } from "webpack";
 import * as WebpackMerge from "webpack-merge";
+*/
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const { Configuration, Plugin } = require("webpack");
+const WebpackMerge = require("webpack-merge");
+
+/*
+export type HarpWebpackEntry = string | { [name: string]: string } |  { [name: string]: string[] };
 
 export interface HarpWebpackConfig {
-    mainEntry?: string;
-    decoderEntry?: string;
+    mainEntry?: HarpWebpackEntry;
+    decoderEntry?: HarpWebpackEntry;
     htmlTemplate?: string;
 }
-
-export function addHarpWebpackConfig(config?: Configuration, harpConfig?: HarpWebpackConfig) {
+export
+*/
+function addHarpWebpackConfig(
+    config /*?: Configuration*/,
+    harpConfig /*?: HarpWebpackConfig*/
+) /*: Configuration[]*/ {
     if (Array.isArray(config) || typeof config === "function") {
         throw new Error("config passed to addHarpWebpackConfig must be a Configuration object");
     }
     const userConfig = config !== undefined ? config : {};
     const mode = process.env.NODE_ENV === "production" ? "production" : "development";
+    const devtool = process.env.PREPARE_ONLY === "true" ? undefined : "source-map";
     const mainEntry = harpConfig === undefined ? undefined : harpConfig.mainEntry;
-    const baseConfig: Configuration = {
+    const baseConfig /*: Configuration*/ = {
         output: {
             filename: "[name].bundle.js"
```

Comment:
General comment: we shouldn't overwrite this particular webpack default (no reason for that).